src/CattyTests/DataModel/VariablesContainerTest.swift

```diff
@@ -96,4 +96,145 @@ final class VariablesContainerTest: XCTestCase {
         XCTAssertEqual(1, container.allLists(for: objectA)?.count)
         XCTAssertEqual(1, container.allLists(for: objectB)?.count)
     }
+
+    func testAllVariables() {
+        let objectA = SpriteObject()
+        objectA.name = "testObjectA"
+
+        let userVariable1 = UserVariable()
+        userVariable1.name = "testName1"
+
+        let userVariable2 = UserVariable()
+        userVariable2.name = "testName2"
+
+        let container = VariablesContainer()
+
+        container.addObjectVariable(userVariable1, for: objectA)
+        var allVariable = container.allVariables() as! [UserVariable]
+
+        XCTAssertEqual(1, allVariable.count)
+        XCTAssertTrue(allVariable[0].name == userVariable1.name)
```

Comment:
Please use `XCTAssertEqual` to compare two strings. In case of a failure, the failed test result will reveal more information compared to `XCTAssertTrue` (stating both the expected and the actual value).